cpp/test/test.cpp

```diff
@@ -64,17 +94,76 @@ void test_user_id_error(void)
 
 void encrypt_decrypt_roundtrip(void)
 {
-    auto s = "{\"accountId\":\"abcABC012_.$#|@/:;=+'-91e078f0-a60c-4251-8652-dd498c07a8f4\",\"segmentId\":1825,\"signingPrivateKey\":\"uKHa70uwLVG3IU7XodT2kla/PuC/En8PkRCjMMc9ZE7HFrOV+g0vOwATp/CiXp65mVas0K6TSl/RaxDGlcmsnA==\",\"devicePrivateKey\":\"YZRlDSkM+JxxSXCtWCVK693qfhNqcbhaPrtHs92uD4w=\"}";
-    DeviceContext d = unwrap(DeviceContext::fromJsonString(s));
+    DeviceContext d = unwrap(DeviceContext::fromJsonString(deviceContextString));
     IronOxide sdk = unwrap(IronOxide::initialize(d, IronOxideConfig()));
     auto encrypted_doc = unwrap(sdk.documentEncrypt(string_to_slice("foo"), DocumentEncryptOpts()));
     auto decrypted = unwrap(sdk.documentDecrypt(vec_to_slice(encrypted_doc.getEncryptedData())));
     TEST_CHECK(vec_to_string(decrypted.getDecryptedData()) == "foo");
     TEST_MSG("Decrypted value is not what was encrypted.");
 }
 
+void group_name(void)
+{
+    auto group_name = unwrap(GroupName::validate("blargh"));
+    TEST_CHECK(group_name.getName().to_std_string() == "blargh");
+}
+
+void group_create_default(void)
+{
+    DeviceContext d = unwrap(DeviceContext::fromJsonString(deviceContextString));
+    IronOxide sdk = unwrap(IronOxide::initialize(d, IronOxideConfig()));
+    auto group_create_result = unwrap(sdk.groupCreate(GroupCreateOpts()));
+    TEST_CHECK(group_create_result.getMemberList().getList().as_slice().size() == 1);
```

Comment:
Add a check for `getAdminList()` too maybe? Although I guess that's just getting down into testing the core of IronOxide itself, so maybe the goal is mostly just "did this pass without segfaulting and return an expected value".

Reply:
This is basically sanity checking, ya. Not really interested in replicating a bunch of tests. Doing the calls is the important bit.